akka-typed-testkit/src/main/scala/akka/typed/testkit/scaladsl/TestProbe.scala

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.testkit.scaladsl
+
+import scala.concurrent.duration._
+import java.util.concurrent.BlockingDeque
+import akka.typed.Behavior
+import akka.typed.scaladsl.Actor._
+import akka.typed.ActorSystem
+import java.util.concurrent.LinkedBlockingDeque
+import java.util.concurrent.atomic.AtomicInteger
+import akka.typed.ActorRef
+import akka.util.Timeout
+import scala.concurrent.Await
+import com.typesafe.config.Config
+import akka.typed.testkit.TestKitSettings
+import akka.util.BoxedType
+import scala.reflect.ClassTag
+
+object TestProbe {
+  private val testActorId = new AtomicInteger(0)
+
+  private def format(u: TimeUnit, d: Duration) = "%.3f %s".format(d.toUnit(u), u.toString.toLowerCase)
+
+  def apply[M]()(implicit system: ActorSystem[_], settings: TestKitSettings): TestProbe[M] =
+    apply(name = "test")
```

Comment:
"testProbe" is good, it must be something different that in classic but there it is "testActor", I think